waffle_configs/bedrock-prod.env

```diff
@@ -10,9 +10,9 @@ FUNNELCAKE_134_PLATFORMS=win,win64
 FUNNELCAKE_135_LOCALES=en-US
 FUNNELCAKE_135_PLATFORMS=win,win64
 FUNNELCAKE_138_LOCALES=en-US
-FUNNELCAKE_138_PLATFORMS=win64
+FUNNELCAKE_138_PLATFORMS=win,win64
```

Comment:
I believe we can remove `win64` here, right? The custom stub installers can now serve the correct build, so unless we want to keep on doing 64bit OS detection on the bedrock side, the easiest thing to do here would be to just serve the regular 32bit funnelcake.